src/lib/spinel/spinel.h

```diff
@@ -1568,7 +1568,15 @@ enum
     SPINEL_PROP_PHY_PCAP_ENABLED   = SPINEL_PROP_PHY__BEGIN + 8,  ///< [b]
     SPINEL_PROP_PHY_CHAN_PREFERRED = SPINEL_PROP_PHY__BEGIN + 9,  ///< [A(C)]
     SPINEL_PROP_PHY_FEM_LNA_GAIN   = SPINEL_PROP_PHY__BEGIN + 10, ///< dBm [c]
-    SPINEL_PROP_PHY__END           = 0x30,
+
+    /// Signal the max power for a channel
```

Comment:
Can you please also update/increment the recently added `SPINEL_RCP_API_VERSION`?

This is a good case of adding new RCP APIs (property) and thinking about whether the host MIN_API_VERSION need to be updated or not. From code above looks like the host would handle the case where there is no such property on RCP.

Reply:
@gjc13, I think the `SPINEL_RCP_API_VERSION` is same as before. Can you please change it? 

We should increment it since there is an RCP related change in spinel (even if it does not break compatibility).  This is similar to the OT API version (which is incremented on any change in public/platform OT APIs, even a comment update/fix).

The policy:
https://github.com/openthread/openthread/blob/d218dfcb354309600c379e307ea4742236f0bacc/src/lib/spinel/spinel.h#L311-L329